app/components/shared/rejection_reasons/reasons_for_rejection_component.html.erb

```diff
@@ -192,70 +154,44 @@
         </li>
       <% end %>
     </ul>
-    <% if editable? %>
-      <p class="app-rejection__actions">
-        <%= govuk_link_to 'Change', provider_interface_reasons_for_rejection_initial_questions_path(application_choice) %>
-      </p>
-    <% end %>
   </div>
 <% end %>
 
 <% if reasons.cannot_sponsor_visa_y_n == 'Yes' %>
   <div class="app-rejection">
-    <%= content_tag(subheading_tag_name, I18n.t('reasons_for_rejection.cannot_sponsor_visa.title'), class: 'govuk-heading-s') %>
+    <h3 class="govuk-heading-s"><%= I18n.t('reasons_for_rejection.cannot_sponsor_visa.title') %></h3>
     <% paragraphs(reasons.cannot_sponsor_visa_details).each do |paragraph| %>
       <p class="govuk-body govuk-!-margin-bottom-3"><%= paragraph %></p>
     <% end %>
-
-    <% if editable? %>
-      <p class="app-rejection__actions">
-        <%= govuk_link_to 'Change', provider_interface_reasons_for_rejection_initial_questions_path(application_choice) %>
-      </p>
-    <% end %>
   </div>
 <% end %>
 
 <% if reasons.why_are_you_rejecting_this_application.present? %>
   <div class="app-rejection">
-    <%= content_tag(subheading_tag_name, I18n.t('reasons_for_rejection.why_are_you_rejecting_this_application.title'), class: 'govuk-heading-s') %>
+    <h3 class="govuk-heading-s"><%= I18n.t('reasons_for_rejection.why_are_you_rejecting_this_application.title') %></h3>
     <% paragraphs(reasons.why_are_you_rejecting_this_application).each do |paragraph| %>
       <p class="govuk-body govuk-!-margin-bottom-3"><%= paragraph %></p>
     <% end %>
-    <% if editable? %>
-      <p class="app-rejection__actions">
-        <%= govuk_link_to 'Change', provider_interface_reasons_for_rejection_other_reasons_path(application_choice) %>
-      </p>
-    <% end %>
   </div>
 <% end %>
 
 <% if reasons.other_advice_or_feedback_y_n == 'Yes' %>
   <div class="app-rejection">
-    <%= content_tag(subheading_tag_name, 'Additional feedback', class: 'govuk-heading-s') %>
+    <h3 class="govuk-heading-s">Additional feedback</h3>
     <% paragraphs(reasons.other_advice_or_feedback_details).each do |paragraph| %>
       <p class="govuk-body govuk-!-margin-bottom-3"><%= paragraph %></p>
     <% end %>
-    <% if editable? %>
-      <p class="app-rejection__actions">
-        <%= govuk_link_to 'Change', provider_interface_reasons_for_rejection_other_reasons_path(application_choice) %>
-      </p>
-    <% end %>
   </div>
 <% end %>
 
 <% if %w[Yes No].include?(reasons.interested_in_future_applications_y_n) %>
   <div class="app-rejection">
-    <%= content_tag(subheading_tag_name, 'Future applications', class: 'govuk-heading-s') %>
+    <h3 class="govuk-heading-s">Future applications</h3>
     <p class="govuk-body">
       <%= I18n.t(
         "reasons_for_rejection.interested_in_future_applications.reason.#{reasons.interested_in_future_applications_y_n.downcase}",
         provider_name: application_choice.provider.name,
       ) %>
     </p>
-    <% if editable? %>
```

Comment:
could we also remove `editable?` from `app/components/shared/rejection_reasons/rejection_reasons_component.html.erb`

Reply:
@cwrw  `editable` is used here https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/views/provider_interface/rejections/check.html.erb#L19 but I think it makes sense to move the `editable?` method into the component specific to the current iteration of R4R, it's not used by the other components now, will update.

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/6876/commits/e0e3d09934b286605a2edc0ff98e31d4a332cb61 should refactor the scope of what is `editable?`